.vscode/settings.json

```diff
@@ -1 +0,0 @@
-{ "eslint.workingDirectories": [{ "mode": "auto" }] }
```

Comment:
this is checked in on purpose to prevent `vscode-eslint` from going haywire. did you have anything you want added to this file?

Reply:
usually not a great practice to check in personal IDE config; given not everyone uses that IDE or uses it the same way